concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2430,6 +2430,29 @@ jobs:
 
                 cf enable-feature-flag diego_docker
 
+      - task: enable-service_instance_sharing-feature-flag
```

Comment:
All we're missing here is some PascalCase :trollface: 

Do you think it'd be worth keeping consistency?

Reply:
I was going with consitency with https://github.com/alphagov/paas-cf/blob/f92d40bfbde04a0a71d1a07c29718800f14f38f1/concourse/pipelines/create-cloudfoundry.yml#L2411